### ISSUE TITLE
Electrum 2.8.3 (Win10) won't sync with 2.8.2 (Android)

### ISSUE THREAD

**Issue by kimmytaz** [spesmilo/electrum#2487]
I have Electrum 2.8.3 on my laptop running windows 10 and Electrum 2.8.2 on my Android phone.
I can see my bitcoins on the phone but nothing is showing on my laptop. I have disabled my firewall but to no avail.
I have enter all the same info on both devices. Same seed, password etc.

Any suggestions?

Mark

**Comment by SomberNight** [comment]
Are you sure Electrum on your laptop successfully connected to a server? What colour is the circle/sphere/orb in the lower right corner (of the GUI)?
Did it previously work on an older version of Electrum on Windows, or is this the first version you tried?
How did you generate the original seed/wallet (what platform and version of Electrum)?
What type of wallet is it? (multi-signature, standard single signer, ...)

**Comment by kimmytaz** [comment]
Hi.
Yes it is connecting to a server. The light is green. I have also tried 
selecting a server and it says connecting and does connect. It says 
"synchronising" in the bottom left of the screen and then shows a 
balance of 0.
This is the first windows version I have used. I installed it the other 
day before I installed the android version on my phone and it showed 
nothing in my bit coin account or history or anything in fact.
When I install the android version it immediately showed my history back 
to 2015 although I only used it twice then.
The original seed/wallet was set up a couple of years ago on a raspberry 
pi but I couldn't login to it as I have forgotten the password to get in 
to my raspberry pi.
My wallet is just a standard one at the moment.

Thanks for your help.

Mark

------ Original Message ------

**Comment by SomberNight** [comment]
Hmm. Would you be willing to come to IRC (#electrum on Freenode) to streamline trying out a few things? We could then post a solution here if we find one.

**Comment by kimmytaz** [comment]
Yeah sure but I can't do it tonight. Are you around on Saturday? 
Otherwise, next week. Also, I've not used IRC Freenode before so you 
might need to give some instructions. I used to work in IT so I'm not a 
novice at PC's so I should pick it up pretty quickly.

Mark

------ Original Message ------

**Comment by SomberNight** [comment]
Any time you pop in, chances are, someone will help you.
By the way, are you sure you have the same wallets on both devices? Could you make sure the addresses generated are the same?

**Comment by kimmytaz** [comment]
Hi. Solved the problem somehow. I created a new wallet using the public 
key from my phone's copy of electrum. This allowed me a watching only 
edition on my laptop in which I could see all my transactions. I then 
created yet another wallet using my existing seed and for some reason I 
can everything now and it should be fully functional rather than just 
watching.

Thanks for you help.

Mark

------ Original Message ------
